tests/compiletest.rs

```diff
@@ -156,9 +163,10 @@ fn main() -> Result<()> {
     // Let the tests know where to store temp files (they might run for a different target, which can make this hard to find).
     env::set_var("MIRI_TEMP", env::temp_dir());
 
-    ui(Mode::Pass, "tests/pass")?;
-    ui(Mode::Panic, "tests/panic")?;
-    ui(Mode::Fail, "tests/fail")?;
+    ui(Mode::Pass, "tests/pass", false)?;
+    ui(Mode::Pass, "tests/pass-crates", true)?;
```

Comment:
I wonder if "pass-dep" would be a name more consistent with how this is called in rustc? It's also called "test_dependencies" after all, not "test_crates".